pages/blog/2022-summary.md

```diff
@@ -0,0 +1,252 @@
+
+### LinkedIn
+
+I'm so damn tired of LinkedIn. Seriously, a platform that drains so much data from others and earns money from job marketing cannot afford and give users access to historical data and see more than one year-old stats. Like seriously...
+
+We had 2139 followers at the end of 2022. Every year's data since account creation looks like below.
+
+<Figure
+  src="/img/posts/2022-summary/linkedin-followers.webp"
+/>
+
+We are not active on LinkedIn. We only republish whatever we publish on Twitter. No engagement is created. We treat it as a news feed, although LinkedIn != Twitter.
+
+Numbers went down a lot compared to last year, but when you look two years back, numbers are very similar. Why? in 2021 we spent $1200 on LinkedIn marketing; thus the number spike for one year.
+
+<Figure
+  src="/img/posts/2022-summary/linkedin-referrals.webp"
+/>
+
+My statement doesn't change. LinkedIn is an important presence, and even without dedicated moderation, simple reposting brings people to the project. We should continue our work there.
+
+### Twitter
+
+Twitter? Seriously, the most unreliable tool ever :smiley: but of course, I'm biased; I do not like it. So let's look at the data. Are people really leaving Twitter because of Elon Musk?
+
+Forget about the number of followers. We have over 3.5k followers, over 500 more than last year. For me, followers number are as reliable as GitHub stars on a repository. Not much...
+
+Let's see how Twitter's presence converts to visits to the AsyncAPI website.
+
+<Figure
+  src="/img/posts/2022-summary/twitter-referrals.webp"
+/>
+
+Exactly the same situation as with LinkedIn. Numbers drop between 2021 and 2022, but again because 2021 was a special year. We experimented that year with paid marketing on Twitter too. So the numbers we should compare are for the years 2020 and 2022. As we can see no change, and it is pretty stable. It looks like Twitter was not really abandoned, at least not for now :smiley:.
+
+My take on it. This is acceptable. I do not think we have to do more there. My observation is that tweets that perform there are not technical tweets that could bring more people to the project, but "thank you" tweets, and "appreciation" tweets. And do not get me wrong, we should still create such tweets, no question about it. It proves it is a social media, not a "technical geeks knowledge exchange center". I do not think any additional investment in the platform makes sense.
+
+## Google Analytics
+
+We use Google Analytics on [AsyncAPI website](https://www.asyncapi.com/) to understand what content is most valuable for you and learn how the interest is growing. The data we get doesn't 100% reflect reality because AsyncAPI is for technical people, and many technical people use plugins in browsers to block  Google Analytics, or browsers that block it natively.
```

Comment:
```suggestion
We use Google Analytics on [AsyncAPI website](https://www.asyncapi.com/) to understand the most valuable content for you and learn how the interest is growing. The data we get doesn't 100% reflect reality because AsyncAPI is for technical people, and many technical people use plugins in browsers to block Google Analytics or browsers that block it natively.
```